phys2bids/io.py

```diff
@@ -209,46 +122,55 @@ def process_acq(channel_list, chtrig, header=[]):
         elif freq_unit == 'kHz':
             freq = freq * 1000
         interval[0] = 1 / freq
-        freq = [freq] * (len(timeseries) + 1)
+        freq = [freq] * len(timeseries)
     else:
         # check if interval is in seconds, if not change the units to seconds and
         # calculate frequency
-        if interval[-1].split('/')[0] != 'sec':
-            LGR.warning('Interval is not in seconds. Converting its value.')
-            if interval[-1].split('/')[0] == 'min':
+        if interval[-1] != 's':
+            LGR.warning('Interval is not in seconds. Converting its value and unit.')
+            if interval[-1] == 'min':
                 interval[0] = float(interval[0]) * 60
                 interval[-1] = 's'
-            elif interval[-1].split('/')[0] == 'msec':
+            elif interval[-1] == 'msec':
                 interval[0] = float(interval[0]) / 1000
                 interval[-1] = 's'
-            elif interval[-1].split('/')[0] == 'µsec':
+            elif interval[-1] == 'µsec':
                 interval[0] = float(interval[0]) / 1000000
                 interval[-1] = 's'
+            if interval[-1] == 'hr':
```

Comment:
```suggestion
           elif interval[-1] == 'hr':
```
I know it's not super mandatory, it's just to have it in the same statement.